Don't have 64 appended to the libdir for 64bit platforms.

JL 05/07/10
Index: beecrypt-4.1.2/configure.ac
===================================================================
--- beecrypt-4.1.2.orig/configure.ac	2010-07-05 16:39:43.000000000 +0100
+++ beecrypt-4.1.2/configure.ac	2010-07-05 16:40:27.521082866 +0100
@@ -585,15 +585,6 @@
 
 # Check where to install the libraries
 bc_libalt=
-case $target_os in
-linux*)
-  case $bc_target_arch in
-  powerpc64 | s390x | x86_64)
-    bc_libalt="64"
-    ;;
-  esac
-  ;;
-esac
 AC_SUBST(LIBALT,$bc_libalt)
 
 # Generate output files.
